python/surf/devices/transceivers/_Qsfp.py

```diff
@@ -124,6 +124,13 @@ def __init__(self,**kwargs):
                 dependencies = [self.TxPwrRaw[2*i+0],self.TxPwrRaw[2*i+1]],
             ))
 
+        self.add(pr.RemoteVariable(
```

Comment:
@antoniobergnoli I created a new PR to cherry pick this new register:
https://github.com/slaclab/surf/pull/1203
Note: Only the lower 4-bits of Register 86 are R/W.  The upper 4-bits are RO. Refer to page8 of URL below
https://docs.broadcom.com/doc/AV02-2098EN